src/store/client-context-provider.jsx

```diff
@@ -28,7 +28,24 @@ export const ClientContextProvider = ({ children }) => {
     setClient(
       new ApolloClient({
         link: authLink.concat(httpLink),
-        cache: new InMemoryCache(),
+        cache: new InMemoryCache({
```

Comment:
Prevents extra fetching, when the cache doesn't know how to merge the 'watched' or 'wishlist' fields.